### ISSUE TITLE
[build] use conan builder

### ISSUE THREAD

**Issue by Horki** [gtrll/gpmp2#41]
@mhmukadam Hi, for easier building, it would be cool to have a `conan` builder.

TODO:
GTSAMCMakeTools; for now, fails and I am not curtained how to fix the path.

```
mkdir -p build && cd build
conan install .. --build=missing
cmake ..
make
```

PS:
What you think?

**Comment by Horki** [comment]
@mhmukadam also gtsam 4.0.2 in conan has dependencies

```
    boost/1.71.0
    bzip2/1.0.8
    eigen/3.3.7
    gtsam/4.0.2
    zlib/1.2.11
```

**Comment by mhmukadam** [comment]
Closed #40 due to inactivity.
